### ISSUE TITLE
Report usages of 'Has.Count.Zero' and propose 'Is.Empty' in NUnit tests

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#487]
When working with NUnit test and assertions, we should detect `Has.Count.Zero` and propose to use `Is.Empty` instead.

Example:
```c#
Assert.That(list, Has.Count.Zero);
```
could be shortened to:
```c#
Assert.That(list, Is.Empty);
```
